### ISSUE TITLE
Code quality: Plan for migration to possible new architecture and for upgrading phpstan levels

### ISSUE THREAD

**Issue by v-stamenova** [HZ-HBO-ICT/it-conference#601]
Example for what issues we'd expect when we introduce phpstan:
https://github.com/v-stamenova/hello-world-website/actions/runs/11765269239/job/32771386351?pr=2

**Comment by TimKardol** [comment]
What do you mean by new architecture?

**Comment by v-stamenova** [comment]
In case the experiment with full livewire front-end works in the SA, we can try to do the same here - because right now the mix between blade and livewire is getting a bit annoying

**Comment by v-stamenova** [comment]
I think since we are using the baseline we won't have to figure out something further? @TimKardol

**Comment by TimKardol** [comment]
Nope. I think we can close this one now that PHPStan has been implemented.
